src/notation/qml/MuseScore/NotationScene/qmldir

```diff
@@ -11,3 +11,4 @@ UndoRedoToolBar 1.0 UndoRedoToolBar.qml
 Timeline 1.0 Timeline.qml
 SelectionFilterPanel 1.0 SelectionFilterPanel.qml
 NotationScrollAndZoomArea 1.0 NotationScrollAndZoomArea.qml
+ScorePropertiesDialog 1.0 ScorePropertiesDialog.qml
```

Comment:
This file should be in the project module, and its name should be ProjectPropertiesDialog